vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -316,29 +328,50 @@ and
                 projectContext.RemoveSourceFile(file)
                 updated <- true
         
-        let updatedRefs = site.AssemblyReferences() |> hashSetIgnoreCase
+        (*
```

Comment:
This will be put back - it's commented out as it's causing issues at the moment.